app/mailers/mailer/notify.rb

```diff
@@ -27,10 +27,35 @@ def monthly_report_commented(comment_id)
       mail(to: target_users.map(&:email), subject: mail_subject(@title))
     end
 
+    def daily_report_registration(user_id, daily_report_id)
+      @report = DailyReport.find(daily_report_id)
+      @user = User.find(user_id)
+      return unless @user.groups.present?
+
+      @title = "#{@user.name}が#{@report.target_date.strftime('%Y年%m月%d日')}の日報を登録しました"
+      mail(to: @user.groups.map(&:email), subject: mail_subject(@title))
+    end
+
+    def daily_report_commented(comment_id)
+      @comment = DailyReportComment.find(comment_id)
+      target_users = daily_report_commented_targets(@comment)
+      return if target_users.blank?
+
+      report = @comment.daily_report
+      report_name = "#{report.user.name}さんの日報（#{report.target_date.strftime('%Y年%m月d日')}）"
+      @title = "#{@comment.user.name}さんが#{report_name}にコメントしました"
+
+      mail(to: target_users.map(&:email), subject: mail_subject(@title))
+    end
+
     private
 
     def monthly_report_commented_targets(comment)
       comment.monthly_report.related_users - [comment.user]
     end
+
+    def daily_report_commented_targets(comment)
```

Comment:
__[reek]__
* [UtilityFunction] doesn't depend on instance state (maybe move it to another class?) [:ref](https://github.com/troessner/reek/blob/master/docs/Utility-Function.md)

<!-- Comment by [SideCI](https://sideci.com) -->